site/get-started.markdown

```diff
@@ -0,0 +1,118 @@
+
+## Running your first lines of code
+
+We have everything set up, let's use it! The Haskell compiler, GHC, comes with an interactive interpreter called GHCi which is great for playing with Haskell and trying things out, so let's give it a spin. Run `ghci` at your command prompt, which will start a new GHCi prompt for you. Let's do a simple calculation to check Haskell's computing capabilities:
+```
+> 6 + 3^2 * 4
+42
+```
+
+42 is a nice even number, but what about the first 10 even numbers after it?
+```
+> take 10 (filter even [43..])
+[44,46,48,50,52,54,56,58,60,62]
+```
+
+What is the sum of those?
+```
+> sum it
+530
+```
+**NOTE**: We used a special feature of GHCi here, which is a special `it` variable that remembers the result of the last expression.
+
+Great, you got the first taste of Haskell! Now let's get to running a real program.
+
+## Writing your first Haskell program
+
+In your editor, create a new file named `hello.hs`. Write the following in it:
+```hs
+main = do
+  putStrLn "Hello, everybody!"
+  putStrLn ("Please look at my favorite odd numbers: " ++ show (filter odd [10..20]))
+```
+
+You can now compile it with `ghc` to produce an executable called `hello` that we will then run:
+```
+> ghc hello.hs
+> ./hello
+Hello, everybody!
+Please look at my favorite odd numbers: [11,13,15,17,19]
+```
+
+There you go, you just wrote a short, polite program in Haskell!
+
+**GHCI TIP**: You can also load your file directly into `ghci`, which will enable you to play with any functions and other definitions you defined in it. So for our example, we can just load `hello.hs` with GHCi and then call the function `main` like this:
+```
+> ghci hello.hs
+GHCi, version 8.10.7: https://www.haskell.org/ghc/  :? for help
+[1 of 1] Compiling Main             ( hello.hs, interpreted )
+Ok, one module loaded.
+> main
+Hello, everybody!
+Please look at my favorite odd numbers: [11,13,15,17,19]
+```
+
+## Participate in the community
+
+By participating in the Haskell community you be able to ask for help and learn about new developments in the Haskell ecosystem. Some of the most popular places to interact with the community are:
+
+ - [Haskell Discourse](https://discourse.haskell.org/)
+ - [r/haskell](https://www.reddit.com/r/haskell/)
+ - [https://wiki.haskell.org/IRC_channel](https://wiki.haskell.org/IRC_channel)
+
+We recommend joining right now, and don't be shy to ask for help! Check [https://www.haskell.org/community](https://www.haskell.org/community) for a full list of resources relating to the Haskell community.
+
+## Next steps
```

Comment:
I'm not entirely sure about this particular list of resources, but I like the general idea of picking a couple of free resources as a starting point.

Reply:
We can discuss it, we had a bit of conversation on it here on the PR and ended up with this. I am big fan of LYAH (even though it is somewhat outdated), and also like the new book with building static blog generator, think they are very accessible and practical. I don't know much about the materials from the uni that we listed but they were on top of course list of Documentation page on haskell.org .
We can modify this if you have better ideas!